src/popup/components/Header.vue

```diff
@@ -107,6 +107,9 @@ export default defineComponent({
     );
 
     function back() {
+      if (root.$route.meta?.directBackRoute) {
```

Comment:
I suggest to move the line 117 before this test and pick the `meta` the same way as the `fullPath` then use the `meta` here and in line 113.

Reply:
good idea